arm-network/2016-12-01/swagger/virtualNetworkGateway.json

```diff
@@ -1600,6 +1611,130 @@
       ],	  
       "description": "Response for GetConnectionSharedKey API service call"
     },
+    "IpsecPolicy": {
+      "properties": {
+        "sALifeTimeSeconds": {
+          "type": "integer",
+          "format": "int64",
+          "description": "The Security Association lifetime in seconds for a site to site VPN tunnel."
+        },
+        "sADataSizeKilobytes": {
+          "type": "integer",
+          "format": "int64",
+          "description": "The Security Association payload size in KB for a site to site VPN tunnel."
+        },
+        "ipsecEncryption": {
+          "type": "string",
+          "description": "The IPSec encryption algorithm (IKE phase 2). Possible values are 'None', 'DES', 'DES3', 'AES128', 'AES192', 'AES256', 'GCMAES128', 'GCMAES192', and 'GCMAES256'.",
```

Comment:
Please don't specify the enum values in the description. Some generators will even auto-gen such description and append it (if appropriate), so this is over-specification. Same applies to other enums below.